assignment-2/src/main/java/dk/jplm/si/assignment2/service/EmailServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package dk.jplm.si.assignment2.service;
+
+import dk.jplm.si.assignment2.model.Guest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EmailServiceImpl implements EmailService {
+
+    @Autowired
+    CountryServiceImp countryService;
+    @Autowired
+    GenderServiceImpl genderService;
+
+    @Override
+    public String sendEmail(String email) throws Exception {
+        throw new Exception("Not implemented yet");
+    }
+
+    private List<Guest> setCountries(List<Guest> guests) {
+        for (Guest g : guests
```

Comment:
This formatting seems a bit strange to me

Reply:
I am not sire what you mean. Can you explain that to me pls?